setup.cfg

```diff
@@ -17,9 +17,9 @@
 [metadata]
 name = Superset
 summary = a data exploration platform
-description-file = README.md
+description_file = README.md
 author = Apache Superset Dev
-author-email = dev@superset.apache.org
+author_email = dev@superset.apache.org
```

Comment:
Bycatch; noticed this was being emitted by `pylint`:

> /Users/ville/apple/apache-superset/.tox/pylint/lib/python3.8/site-packages/setuptools/dist.py:770: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead
>   warnings.warn(
/Users/ville/apple/apache-superset/.tox/pylint/lib/python3.8/site-packages/setuptools/dist.py:770: UserWarning: Usage of dash-separated 'author-email' will not be supported in future versions. Please use the underscore name 'author_email' instead